custom/src/main/java/com/splunk/opentelemetry/InstrumentationLibraryTracerCustomizer.java

```diff
@@ -36,11 +39,22 @@ private static boolean spanProcessorInstrumentationLibraryEnabled() {
     return ("true".equalsIgnoreCase(value));
   }
 
+  private static boolean jdbcSpanLowCardinalityNameEnabled() {
+    String value = System.getProperty(ENABLE_JDBC_SPAN_LOW_CARDINALITY_NAME_PROPERTY);
+    if (value == null) {
+      value = System.getenv(propertyToEnv(ENABLE_JDBC_SPAN_LOW_CARDINALITY_NAME_PROPERTY));
+    }
+    // enabled by default
+    return value == null || "true".equalsIgnoreCase(value);
```

Comment:
How about just `Boolean.valueOf(value)`. I think the logic is the same.

Reply:
Pretty close, but unfortunately not the same:

```java
    public static Boolean valueOf(String s) {
        return parseBoolean(s) ? TRUE : FALSE;
    }

    public static boolean parseBoolean(String s) {
        return "true".equalsIgnoreCase(s);
    }
```

Reply:
so the difference is? From the javadoc, it looks like the same thing to me.

Reply:
Oh, I just realized that you meant just the  `"true".equalsIgnoreCase(value)` part, not the entire line 🤦‍♂️

Reply:
In that case, yes, you're exactly right -- I'll change that.

Reply:
right.  :). sorry for the confusion.